fastlane/docs/Actions.md

```diff
@@ -2438,6 +2438,12 @@ Install an Xcode plugin for the current user
 install_xcode_plugin(url: 'https://github.com/contentful/ContentfulXcodePlugin/releases/download/0.5/ContentfulPlugin.xcplugin.zip')
```

Comment:
Let's use a different example for the `url` then, so that it's clear to not use this for GitHub links